microsoft-365/managed-desktop/service-description/DeviceRegistration.md

```diff
@@ -125,38 +138,19 @@ You can monitor the progress of device registration on the main **Microsoft Mana
 
 [//]: # (James and Jose are working on changing the Error Codes to be better.)
 
-<table>
-    <tr>
-        <td>Error Message</td>
-        <td>Details</td>
-    </tr>
-    <tr>
-        <td>Unexpected Error</td>
-        <td>Your request could not be automatically processed. Please contact Support <link to Ops page> and provide the Request ID from your error export.</td>
-    </tr>
-    <tr>
-        <td>Invalid Hardware Hash</td>
-        <td>The hardware hash you provided for this device was incorrectly formatted. Please verify the hardware hash <link to docs> and resubmit. </td>
-    </tr>
-    <tr>
-        <td>Device already registered</td>
-        <td>This device is already registered to your company. No further action required.</td>
-    </tr>
-    <tr>
-        <td>Device not found</td>
-        <td>We couldn’t deregister this device because it does not exist in your company. No further action required. </td>
-    </tr>
-    <tr>
-        <td>Device claimed by another company</td>
-        <td>This device has already been claimed by another company. Please check with your device supplier. </td>
-    </tr>
-    <tr>
-        <td>Device not found</td>
-        <td>We couldn’t register this device because we could not find a match for the provided Manufacturer, Model, and Serial Number. Please confirm the values with your device supplier.</td>
-    </tr>
-    </table>
+| Error | Details |
```

Comment:
Thanks for this. From now on I'll use the MD style tables - appreciate the translation this time.

Reply:
No worries! HTML can hork things up, so we don't use it unless there is just no other option (and there are cases where that is true).